scripts/search-for-rhc.mjs

```diff
@@ -7,13 +7,32 @@
 
 import { $, argv } from 'zx';
 
-const buildFolder = argv._[0];
-console.log(`🔍 Inspecting folder: ${buildFolder}`);
-const flags = ['--recursive', '--quiet', '--include=*.js', '--include=*.mjs'];
-
-// Leveraging https://google.github.io/zx/process-promise#nothrow to avoid throwing an error if the command fails
-if ((await $`grep ${flags} 'https://\${scriptHost}/npm/@clerk/clerk-js' ${buildFolder}`.exitCode) === 0) {
-  throw new Error('Found RHC in build output');
-} else {
-  console.log('✅ No RHC found in build output');
+const targetType = argv._[0]; // file | directory
+const target = argv._[1]; // Target of the resource
+
+async function asyncSearchRHC(name, search) {
+  const cmd = () =>
+    targetType === 'directory'
+      ? $`grep -rFq --include=\\*.js --include=\\*.mjs "${search}" ${target}`
```

Comment:
@tmilewski As a note in case we want to change it back:

- I've used the longer forms of the flags because then you don't have to look up what `-rFq` does
- `zx` has a feature where you can pass an array of flags to the command and it escapes everything already, that's why `${flags}` was used

Reply:
@LekoArts Yeah, we can move to the longer forms. 

My original goal was to have `zx` handle flags for in that manner, but for some reason, it simply wasn’t parsing things correctly. This seems fine for the time being as it’s quite minimal.